src/clean-ast.js

```diff
@@ -148,6 +149,13 @@ function massageAST(ast) {
 
       quasis.forEach(q => delete q.value);
     }
+    // styled-components
+    if (
+      ast.type === "TaggedTemplateExpression" &&
+      ast.tag.type === "MemberExpression"
+    ) {
```

Comment:
This is not enough. Relay.QL would match. I think you need to check if the left is `style` or the right is `extends`.